website/www/site/content/en/blog/beamquest.md

```diff
@@ -0,0 +1,37 @@
+---
+title:  "Get Started with Apache Beam: An Open Source Quest sponsored by Google Cloud"
+date:   2023-06-09 00:00:01 -0800
+categories:
+  - blog
+aliases:
+  - /blog/2023/06/09/beam-quest.html
+authors:
+  - svetaksundhar
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### **What is the quest?**
+[Get Started with Apache Beam Quest](https://www.cloudskillsboost.google/quests/310) is a series of four qwiklabs that test proficiency in Apache Beam. Upon completion, a sharable credential badge is granted!
+
+
+### **Apache Beam Playground Features**
```

Comment:
```suggestion
### **Apache Beam playground features**
```

Reply:
this wasn't related to my change -- should I fix anyway?

Reply:
You don't need to change it.